projects/fix-machos.com/package.yml

```diff
@@ -0,0 +1,39 @@
+versions:
+  - 1.0.0
+
+# platforms: [darwin] # not used by our CI, sadly
+
+build:
+  dependencies:
+    zlib.net: ^1
+  script:
+    - mkdir -p "{{prefix}}"/{bin,lib}
+
+    # none of these hacks do anything on linux
+    - run: exit 0
+      if: linux
+
+    - run: |
+        clang -o '{{prefix}}/bin/fix-machos-test' $PROP -lz
+      prop:
+        content: |
+          #include <stdio.h>
+          int main() {
+            printf("Hello, world!\n");
+            return 0;
+          }
+        extname: c
+
+    - cp {{deps.zlib.net.prefix}}/lib/libz.dylib '{{prefix}}/lib'
+
+    - run: install_name_tool -change @rpath/zlib.net/v{{deps.zlib.net.version}}/lib/libz.{{deps.zlib.net.version.marketing}}.dylib {{prefix}}/lib/libz.dylib fix-machos-test
```

Comment:
why this line?

Reply:
to ensure we have a linked library with the build prefix.